src/utils/artifact/generateAragonArtifact.ts

```diff
@@ -50,40 +46,45 @@ function _generateAragonArtifact(
     abi,
     // Additional metadata
     flattenedCode: './code.sol',
-    appName,
-    appId: appName ? getAppId(appName) : ''
+    appName: appEnsName,
+    appId: getAppId(appEnsName)
   }
 }
 
 // Function Overloading logic
 
 /**
  * Returns aragon artifact.json from app data
+ * @param appEnsName "finance.aragonpm.eth"
  * @param arapp
  * @param abi
  * @param functions Parsed contract function info
  */
 export function generateAragonArtifact(
+  appEnsName: string,
   arapp: AragonAppJson,
   abi: AbiItem[],
   functions: AragonContractFunction[]
 ): AragonArtifact
 
 /**
  * Returns aragon artifact.json from app data
+ * @param appEnsName "finance.aragonpm.eth"
  * @param arapp
  * @param abi
  * @param flatCode Flat code of target contract plus all imports
  * @param contractName Target contract name or path: "Finance" | "contracts/Finance.sol"
  */
 export function generateAragonArtifact(
+  appEnsName: string,
   arapp: AragonAppJson,
   abi: AbiItem[],
   flatCode: string,
```

Comment:
Maybe we can rename this to "source code" following the convention below?

Reply:
I advised Gabi against since it MUST be the flat code not just the main contract code. If you do flatSourceCode or something that clearly stays it's more than one contract's code then ok

Reply:
Perhaps we can rename the following function's parameter name then

Reply:
Thanks for picking it up! Added as a todo in https://github.com/aragon/buidler-aragon/issues/131